ArduCopter/defines.h

```diff
@@ -107,12 +107,12 @@ enum GuidedMode {
 };
 
 // Safe RTL states
-enum SmartRTLState {
-    SmartRTL_WaitForPathCleanup,
-    SmartRTL_PathFollow,
-    SmartRTL_PreLandPosition,
-    SmartRTL_Descend,
-    SmartRTL_Land
+enum  class SmartRTLState {
+    WaitForPathCleanup,
```

Comment:
Should be caps.  If you wanted us to review this policy of `MixedCase::UPPERCASE` enum classes at a devcall, do let us know :-)